src/run.js

```diff
@@ -152,15 +154,16 @@ const processPage = ({
 
       await page.setRequestInterception(true)
       page.on('request', request => {
-        if (/data:image\//.test(request.url())) {
+        const requestUrl = request.url()
```

Comment:
I just noticed we used `request.url()` so many times that it felt right to make a local variable.